### ISSUE TITLE
Project abandoned

### ISSUE THREAD

**Issue by vinyll** [bosonic/bosonic#32]
Sadly this project seems to be abandoned.
I was glad to find a close to web components standards library, simple, with HTML imports, just doing its job and doing it right.

Polymer is an overkill solution with a heavy environment.
React has a non standard JSX approach.
Riot has too fancy features.
x-tags is full JS with no `<template>` nor `<style>` within components.
VueJS relies on heavy solutions such as WebPack.

I guess I will need continue my quest.
Thank you for having built this, it was a very, very promising and interesting project!

:wave: :cupid:

**Comment by eavichay** [comment]
@vinyll Try slim.js http://slimjs.com https://github.com/slimjs/slim.js

**Comment by vinyll** [comment]
I moved to http://riot.js.org in the end which has gone closer to the W3C standard webcomponents and is still very simple.

**Comment by eavichay** [comment]
@vinyll Riot is lightweight but it is a compiler, it is not exactly "close to webcomponents standard". The generated code is far from being a custom element. Could you share the project you are working on?

**Comment by vinyll** [comment]
I say "close to webcomponent standards" in a sense of including an HTML portion from a file and writing CSS in CSS and HTML in HTML vs JSX-like.

If you're curious about the project I'm working on, it's published on https://github.com/betagouv/eac (with [a demo available](https://betagouv.github.io/eac/index.html)).

**Comment by goldoraf** [comment]
@vinyll Yeah, I abandoned this project 2 years ago because I failed to get people interested about Bosonic and web components. I still believe this is the way to go for "standard" UI components, but I didn't want to loose more time on a project without a community.

Thanks for your very kind words though, I'm happy to know at least some people found the project interesting ;)

**Comment by eavichay** [comment]
@goldoraf I am pretty impressed with your project, I would like to invite you to collaborate on slim.js.
The aim is for use simplicity, which I believe is more important than feature richness.

It is a base class that supports data binding out of the box, without mixins, and with the ability to be enhanced with custom "directives" or plugins.

WDYT?

**Comment by goldoraf** [comment]
@eavichay thanks a lot for your proposition, but TBH, my goal wasn't to build a library that helps building web components, but to build a UI library of ready-to-use web components, whose code would be as closest to standards as possible. If you want to go that way with slim.js, I'd happy to try to migrate Bosonic components to slim.js, but be advised it's a considerable job to build such a library.

**Comment by eavichay** [comment]
@goldoraf That is exactly the roadmap i am planning. Various ready to use components based on slim.js. I have found that slim.js is very fast and the reason I built it is because Polymer requires too much boilerplate to get started. Consider joining forces, I would be happy to create a community around this project.
